AUTHORS.rst

```diff
@@ -18,7 +18,8 @@ Documentation Contributors
 - Eric Woolard `@ericwoolard <https://github.com/ericwoolard>`_
 - Marc `@PerhapsSomeone <https://github.com/PerhapsSomeone>`_
 - kungming2 `@kungming2 <https://github.com/kungming2>`_
-- Add "Name <email (optional)> and github profile link" above this line.
+- PythonCoderAS `@PythonCoderAS <https://github.com/PythonCoderAS>`_
```

Comment:
Can you just add yourself to source contributor? Feel free to remove anyone from documentation contributor that is also a source contributor.

Reply:
Ok.

Reply:
What about kungming?

Reply:
> Feel free to remove anyone from documentation contributor that is also a source contributor.

Reply:
I removed him from there but kept him as logo creator